src/components/AppModal.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="container" :class="{ inactive: !isModalActive }">
```

Comment:
Rather use a directive for modal rendering here, not css classes
you can use `v-if`